store.html

```diff
@@ -42,34 +40,62 @@ <h1 class="text-param">Order your Karma WiFi device today!<h1/>
                     </div>
                     <form>
                         <fieldset>
+                            <!-- details -->
+                            <div class="form-dimensions">
                                 <div class="full-name">
                                     <div class="form-elem">
-                                        <label for="first__name">First Name*</label>
-                                        <input type="text" name="first-name" id="first__name"/>
+                                        <label for="first__name">First Name *</label>
+                                        <input type="text" name="first-name" id="first__name" required="true"/>
                                     </div>
                                     <div class="form-elem">
-                                        <label for="last__name">Last Name*</label>
-                                        <input type="text" name="last-name" id="last__name"/>
+                                        <label for="last__name">Last Name *</label>
+                                        <input type="text" name="last-name" id="last__name" required="true"/>
                                     </div>
                                 </div>
                                 <div class="form-elem">
-                                    <label for="address">Address*</label>
-                                    <input type="text" name="address" id="address"/>
+                                    <label for="address">Address *</label>
+                                    <input type="text" name="address" id="address" required="true"/>
                                 </div>
                                 <div class="form-elem">
                                     <label for="address__2">Address 2</label>
                                     <input type="text" name="address-2" id="address__2" />
                                 </div>
                                 <div class="full-name">
                                     <div class="form-elem">
-                                        <label for="city">City*</label>
-                                        <input type="text" name="city" id="city"/>
-                                    </div>
+                                        <label for="city">City *</label>
+                                        <select class="box" name="city" id="city" required="true">
+                                            <option selected>Select your City...</option>
```

Comment:
You can add multi-options to select like https://www.w3schools.com/tags/tag_select.asp